### PR TITLE
Add style option

### PR DESCRIPTION
**Pull request by myklt**
Used mainly for `antd` integration. Idea based on `babel-plugin-import`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "swc-plugin-transform-import",
-  "version": "1.0.1",
+  "name": "@myklt/swc-plugin-transform-import",
```
Comment (ankitchouhan1020) [inline package.json]: Please fix the package name and version.

**Review by ankitchouhan1020 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (myklt) [reply package.json]: Done. Sorry, completely forgot about it as I was testing with private package.

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
-    "outDir": "lib"
+    "outDir": "lib",
+    "target": "es6"
```
Comment (myklt) [inline tsconfig.json:5]: Unfortunately I cannot upgrade to newer Node.js yet, so changed the target. Is it fine with you?

Comment (ankitchouhan1020) [reply tsconfig.json:5]: Cool, It will work.

**Review by ankitchouhan1020 — APPROVED** [review]

**Comment by ankitchouhan1020** [comment]
Thanks @myklt 🙌 Merged.

**Pull request merged**